plutus-core/testlib/PlutusCore/Generators/QuickCheck/Builtin.hs

```diff
@@ -244,7 +244,9 @@ instance Arbitrary Data where
     arbitrary = arbitraryBuiltin
     shrink = shrinkBuiltin
 
-deriving via Map ByteString (Map ByteString Integer) instance Arbitrary Value
+instance Arbitrary Value where
+    arbitrary = Value.pack <$> arbitrary
+    shrink = fmap Value.pack . shrink . Value.unpack
```

Comment:
This isn't a good shrinker, I have a blog post about [that](https://github.com/effectfully-ou/sketches/tree/master/better-counterexample-minimization). Not too important though.